assets/js/book.js

```diff
@@ -0,0 +1,14 @@
+// Book Constructor
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+  Book.count++;
+  // Generate randomized unique ID for every new book
+  this.id = Math.round(Math.random() * 1000000000);
```

Comment:
This is a Hack and if by chance we end up with two elements having the same id there's going to be trouble. What happened to the static variable?

Reply:
Yup we just wanted to minimize any access to the Book.count. . But yeah you're right :) We just couldn't figure out how to make Book.count private